src/components/TouristInfoCards.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+export default function TouristInfoCards() {
+  return (
+    <div className="cont-card">
+      <div className="card">
+        <img
```

Comment:
Looking good here! For further refactors, you have duplication of the divs with `className="card"` ( except for the actual info they display ). You can further abstract out this div into its own component and then render that component in here for every tourist info card you've got.